chromium_src/chrome/browser/notifications/notification_handler.h

```diff
@@ -24,7 +24,8 @@ class NotificationHandler : public NotificationHandler_ChromiumImpl {
     SHARING = 6,
     ANNOUNCEMENT = 7,
     NEARBY_SHARE = 8,
-    BRAVE_ADS = 9,
+    NOTIFICATIONS_MUTED = 9,
+    BRAVE_ADS = 10,
```

Comment:
is it safe to change? Maybe leave it alone (i.e. `= 9`) and allow google to grow the list further?

Reply:
We shifted it a few times before, but it might be safer to do the way you suggest.